### ISSUE TITLE
Git 2.27.0 64-bit on Windows Server 2012: Git Bash not running and produces mintty.exe.stackdump

### ISSUE THREAD

**Issue by BuddyButterfly** [git-for-windows/git#2761]
- [X] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Windows Server 2012 64bit

```
$ git --version --build-options

git version 2.27.0.windows.1
cpu: x86_64
build from commit: 907ab1011dce9112700498e034b974ba60f8b407
sizeof-long: 4
sizeof-sizet_t: 8

```
```
$ cmd.exe /c ver

Microsoft Windows [Version 6.2.9200]
```

 - Options chosen: Defaults, apart from editor nano and no experimental stuff.

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

Editor Option: Nano
Custom Editor Path:
Path Option: Cmd
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Git Pull Behavior Option: Merge
Performance Tweaks FSCache: Enabled
Use Credential Manager: Enabled
Enable Symlinks: Disabled
Enable Pseudo Console Support: Disabled

```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

n.a.

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

Bash

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
n.a.

```
 - What did you expect to occur after running these commands?

Bash to start ;-) 

 - What actually happened instead?

Bash did not start ;-)

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

n.a.

**Comment by BuddyButterfly** [comment]
- mintty.exe.stackdump

Exception: STATUS_ACCESS_VIOLATION at rip=7FEAE0D9CC4
rax=0000000000000000 rbx=0000000000000002 rcx=00000000FFFFC420
rdx=0000000000000001 rsi=00000000FFFFC500 rdi=0000000000000001
r8 =0000000000000002 r9 =0000000000000000 r10=0000000000000000
r11=00000000FFFFC440 r12=0000000080000000 r13=00000001004B24CC
r14=00000000FFFFC4C0 r15=00000001004C8780
rbp=00000000FFFFC4E0 rsp=00000000FFFFC3D8
program=C:\Program Files\Git\usr\bin\mintty.exe, pid 509, thread main
cs=0033 ds=002B es=002B fs=0053 gs=002B ss=002B
Stack trace:
Frame        Function    Args
000FFFFC4E0  7FEAE0D9CC4 (00822000000, 000FFFFC4C0, 00000CF0000, 00000000000)
000FFFFC4E0  7FEAB3DF67D (7FEAD8C3625, 00000000232, 00000000002, 00000000000)
000FFFFC4E0  7FEA9EC9904 (0010043DEF0, 000FFFFC580, 00000000019, 000FFFFC890)
000FFFFC4E0  7FEA9EC6DF0 (00080000000, 00800043D40, 00000000125, 00000000001)
000FFFFC580  0010044EB20 (000FFFFCC8F, 000FFFFCC91, 001800598D0, 001802E7100)
000FFFFCCE0  0018004AEAA (00000000000, 00000000000, 00000000000, 00000000000)
000FFFFFFF0  00180048846 (00000000000, 00000000000, 00000000000, 00000000000)
000FFFFFFF0  001800488F4 (00000000000, 00000000000, 00000000000, 00000000000)
End of stack trace

**Comment by BuddyButterfly** [comment]
Issue solved with git version 2.28.0-rc2-64-bit!
